lib/Parser/ptree.h

```diff
@@ -600,9 +600,21 @@ struct PnFinally : PnStmt
     ParseNodePtr pnodeBody;
 };
 
+enum JsType : byte 
+{
+	t_object,
```

Comment:
I think we should add a Unknown/None type, to tell if we didn't get an annotation for this node